Formula/t/tbb.rb

```diff
@@ -27,37 +26,26 @@ def python3
   end
 
   def install
-    # Prevent `setup.py` from installing tbb4py directly into HOMEBREW_PREFIX.
-    # We need this due to our Python patch.
-    site_packages = Language::Python.site_packages(python3)
-    inreplace "python/CMakeLists.txt", "install --prefix build -f", "\\0 --install-lib build/#{site_packages}"
-
-    tbb_site_packages = prefix/site_packages/"tbb"
-    ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath},-rpath,#{rpath(source: tbb_site_packages)}"
```

Comment:
And this too, but perhaps only for the Python build.

Reply:
fixed